apps/utils/deletion.py

```diff
@@ -133,3 +134,40 @@ def _get_m2m_related_models(model):
             through_model = field.through if hasattr(field, "through") else field.remote_field.through
             m2m_models[through_model] = field.related_model
     return m2m_models
+
+
+def get_related_objects(instance, pipeline_param_key: str = None) -> list:
```

Comment:
Just wondering if this is the best file to put these utils?

Reply:
I'm open to moving them - got a suggestion?

Reply:
Hmm, maybe the file should be renamed instead? `model_relations`?  (let's see if this works: @coderabbit can you suggest a suitable name?)

Reply:
`relations.py` seems suitable to me. I realize I'm being nit picky here. @snopoke up to you